esp-radio/src/ble/os_adapter_esp32c6.rs

```diff
@@ -59,12 +59,25 @@ pub(crate) static BLE_CONFIG: esp_bt_controller_config_t = esp_bt_controller_con
     cca_low_tx_pwr: 0, //???
     main_xtal_freq: 40,
     ignore_wl_for_direct_adv: 0,
-    config_magic: 0x5A5AA5A5,
-
     cpu_freq_mhz: 160,
     enable_pcl: 0, // CONFIG_BT_LE_POWER_CONTROL_ENABLED
     version_num: 0,
     csa2_select: 1,
+    enable_csr: 0,
+    ble_aa_check: 0,
+    ble_llcp_disc_flag: 0b111, /* (BT_CTRL_BLE_LLCP_CONN_UPDATE | BT_CTRL_BLE_LLCP_CHAN_MAP_UPDATE | BT_CTRL_BLE_LLCP_PHY_UPDATE) */
```

Comment:
Should we introduce named constants? Or this is only touched when someone updated the blobs so that would actually make the process worse?

Reply:
Once things are configurable, we should have constants or an EnumSet